bundles/org.openhab.binding.astro/README.md

```diff
@@ -145,7 +145,7 @@ astro:sun:home  [ geolocation="52.5200066,13.4049540,100", interval=60 ]
 astro:moon:home [ geolocation="52.5200066,13.4049540", interval=60 ]
 ```
 
-or optionally with an event offset
+or optionally with an event offset (this can be used in rules triggers)
```

Comment:
This part is about the thing declaration. It has nothing to do with triggers itself. 
If you want to clarify how the event / trigger channel (with or without the offset) can be consumed, i would suggest to add some notes at line 97.

Reply:
I've reverted that part, it's spelled out on line 199